pkg/kubectl/describe.go

```diff
@@ -1882,6 +1886,59 @@ func (p *PetSetDescriber) Describe(namespace, name string, describerSettings Des
 	})
 }
 
+type CertificateSigningRequestDescriber struct {
+	client *client.Client
+}
+
+func (p *CertificateSigningRequestDescriber) Describe(namespace, name string, describerSettings DescriberSettings) (string, error) {
+	csr, err := p.client.Certificates().CertificateSigningRequests().Get(name)
+	if err != nil {
+		return "", err
+	}
+
+	cr, err := utilcertificates.ParseCertificateRequestObject(csr)
+	if err != nil {
+		return "", fmt.Errorf("Error parsing CSR: %v", err)
+	}
+
+	return tabbedString(func(out io.Writer) error {
+		fmt.Fprintf(out, "Name:\t%s\n", csr.Name)
+		fmt.Fprintf(out, "Labels:\t%s\n", labels.FormatLabels(csr.Labels))
+		fmt.Fprintf(out, "Annotations:\t%s\n", labels.FormatLabels(csr.Annotations))
+		fmt.Fprintf(out, "CreationTimestamp:\t%s\n", csr.CreationTimestamp.Time.Format(time.RFC1123Z))
+		fmt.Fprintf(out, "RequestingUser:\t%s\n", csr.Spec.Username)
+		fmt.Fprintf(out, "CommonName\t%s\n", cr.Subject.CommonName)
```

Comment:
should we include all the components of the subject? those could be significant if the approver issues a client certificate for this CSR (c.f. https://github.com/kubernetes/kubernetes/issues/30260)